includes/notifications/controllers/class.llms.notification.controller.failed.attempts.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Notification Controller: Failed Attempts
+ *
+ * @package LifterLMS/Notifications/Controllers/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Notification Controller: Failed Attempts.
+ */
+class LLMS_Notification_Controller_Failed_Attempts extends LLMS_Abstract_Notification_Controller {
+
+	/**
+	 * Trigger Identifier.
+	 *
+	 * @since [version]
+	 * @var [type]
+	 */
+	public $id = 'failed_attempts';
+
+	/**
+	 * Number of accepted arguments passed to the callback function.
+	 *
+	 * @var int
+	 */
+	protected $action_accepted_args = 2;
+
+	/**
+	 * Action hooks used to trigger sending of the notification.
+	 *
+	 * @var array
+	 */
+	protected $action_hooks = array( 'lifterlms_failed_attempts' );
+
+	/**
+	 * Callback function called when a student failed in all quiz attempts.
+	 *
+	 * @param int $student_id WP User ID of a LifterLMS Student
+	 * @param int $course_id  WP Post ID of a LifterLMS Course
+	 * @param int $quiz_id    WP Post ID of a LifterLMS Quiz
+	 *
+	 * @since [version]
+	 * @version [version]
+	 * @return void
```

Comment:
you know the correct order, and that we don't use `@version`

Reply:
yeah, sorry I forgot it. Fixed now. 🤞